marklogic-data-hub/src/main/java/com/marklogic/hub/deploy/commands/DeployDatabaseFieldCommand.java

```diff
@@ -80,7 +80,7 @@ protected String addExistingFieldsAndRangeFieldIndexes(String payload, DatabaseM
 
         addExistingFields(newProps, existingProps);
         addExistingRangeFieldIndexes(newProps, existingProps);
-
+        addExistingRangePathIndexes(newProps, existingProps);
```

Comment:
Good catch - there's an existing unit test that you can easily add setup and assertions to as well for this.

Reply:
Updated the unit test to check the path range indexes as well.